lib/web_ui/dev/felt.dart

```diff
@@ -17,12 +17,15 @@ CommandRunner runner = CommandRunner<bool>(
   ..addCommand(TestsCommand());
 
 void main(List<String> args) async {
+  if (args.isEmpty) {
+    // The felt tool was invoked with no arguments. Print usage.
+    runner.printUsage();
+    io.exit(64); // Exit code 64 indicates a usage error.
+  }
+
   try {
     final bool result = await runner.run(args);
-    if (result == null) {
-      runner.printUsage();
-      io.exit(64); // Exit code 64 indicates a usage error.
-    } else if (!result) {
+    if (result == false) {
       print('Command returned false: `felt ${args.join(' ')}`');
```

Comment:
Actually, it's not the `felt ARGS` that returned false but a sub-command, so more like `Sub-command returned false: ${args.join(' '}`